docs/components/concepts/outbound-connectors-job-workers.md

```diff
@@ -0,0 +1,64 @@
+
+If they both share the same core functionality, how do they differ, and when should you choose what? Connectors and job workers serve different purposes when it comes to aspects like delivery, reusability, focus, and context.
+
+## Delivery
+
+A Connector is reusable code, written as an `OutboundConnectorFunction` using the [Connector SDK](/components/connectors/connector-sdk.md#runtime-logic).
+It is not a standalone application, you cannot start it and have it work on Camunda Platform 8 jobs.
+Instead, a Connector is delivered as a library and can be used in combination with other Connectors in a [Connector runtime environment](/components/connectors/connector-sdk.md#runtime-environments).
+
+In contrast, a job worker is usually part of a Zeebe Client application that can be directly executed to work on jobs.
+
+## Reusability
+
+A job worker usually runs as or inside a standalone Zeebe Client. Without effort, you cannot simply run this in Camunda Platform 8 SaaS or any other environment.
+As a Self-Managed user, you can run it standalone, but often not directly reuse the logic in your existing Zeebe Client that you might already have.
+You can manually extract the job handler from the given job worker, but you also have to ensure that it is still working as expected afterward.
+
+In contrast, a Connector itself is environment-agnostic. There is a runtime environment for Camunda Platform 8 SaaS that can wrap and call this Connector. As the Connector developer, you don't have to worry about this as the runtime takes care of it if you developed the Connector using the Connector SDK.
+
+You can also run the exact same Connector (without any modification) in Camunda Platform 8 Self-Managed; either as a standalone job worker, as additional job handler in your existing Zeebe Client application, or together with other Connectors in one Zeebe Client application.
+This all comes with the Connector SDK, and there is no additional code necessary to get started. However, if you need a custom environment, the Connector SDK provides a guide and default helpers to do that.
+
+## Focus
+
+A job worker is often a complete Zeebe Client application, dealing with environment tasks like handling variables in and out. The core logic of calling a defined URL is only part of the application.
+Plus, it handles Camunda Platform 8-specific APIs like the job worker API to handle variables, complete executions, and throw errors.
+
+A Connector only consists of core business functionality. No environment tasks, no Camunda Platform 8 job worker-related code. You can run this from Camunda Platform 7 as well, if you have a runtime that takes care of this.
+The Connector only needs input variables and access to secrets so they can be used in defined input attributes.
```

Comment:
Technically speaking, access to secrets is optional. If there's no provider configured, secrets will be fetched from operating system environment variables. But I am not sure if we need to be that specific here.

Reply:
@akeller added these tasks here https://github.com/camunda/team-connectors/issues/386